src/qibo/backends/abstract.py

```diff
@@ -66,7 +66,7 @@ def cast(self, x, copy=False):  # pragma: no cover
         raise_error(NotImplementedError)
 
     @abc.abstractmethod
-    def issparse(self, x):  # pragma: no cover
+    def is_sparse(self, x):  # pragma: no cover
```

Comment:
I'm not using this myself, and I expect no user is directly using it. But did you scan all examples and other relevant locations (i.e. Qibojit) to make sure this won't break anything?

Reply:
I did it here but not in `qibojit`. I'll create the corresponding PR there.